### ISSUE TITLE
[BUG] File name after download doesn't match the PAPERLESS_FILENAME_FORMAT variable

### ISSUE THREAD

**Issue by bluemangotree** [paperless-ngx/paperless-ngx#1448]
### Description

If I download a file via UI the used file name for the document doesn't match the variable selected: PAPERLESS_FILENAME_FORMAT

### Steps to reproduce

1. Open a document
2. Click on Download
3. Check the file name of your document

### Paperless-ngx version

1.8.0

### Host OS

Cloudron

### Installation method

Other (please describe above)

### Browser

Chrome

### Configuration changes

PAPERLESS_FILENAME_FORMAT={created_year}{created_month}{created_day}_{title}

**Comment by stumpylog** [comment]
I don't think it's intended to follow the filename format, especially given that format can include folders.  If something in the documentation indicates it will follow the format, that should be fixed.

**Comment by shamoon** [comment]
Yes this is not expected behavior, we can call this a feature request
